### PR TITLE
Remove a rogue raw reference to env var

### PR DESCRIPTION
**Pull request by prprabhu-ms**
Missed in my original PR #2490

### CODE REVIEW

**Review by prabhjot-msft — APPROVED** [review]

File: packages/react-composites/scripts/runBrowserTests.mjs [hunk 1]
```diff
@@ -128,13 +150,48 @@ async function runOne(args, composite, hermeticity) {
...
+  fs.writeFileSync(path.join(PREPROCESSED_ROOT, 'tsconfig.json'), '{ "extends": "../tsconfig.preprocess.json" }');
+}
+
+async function preprocessDir(fromDir, toDir) {
+  console.log(`Preprocessing ${fromDir} to ${toDir}`);
+  fs.mkdirsSync(toDir);
+  fs.copySync(fromDir, toDir, { overwrite: true, preserveTimestamps: true });
+  await exec(
+    quote([
+      'npx',
```
Comment (PorterNan) [inline packages/react-composites/scripts/runBrowserTests.mjs:180]: Maybe we can execute command here from package.json file like `npm run preprocess: test` or so (with 2nd parameter)? Good to have a centralized place to store all the commands (I can feel some other reasons we put command here as well, so highly depends on you)

File: packages/react-composites/tests/browser/call/hermetic/CallReadiness.test.ts [hunk 2]
```diff
@@ -6,21 +6,19 @@ import { expect, Page } from '@playwright/test';
...
   isTestProfileMobile,
-  isTestProfileStableFlavor,
   pageClick,
   stableScreenshot,
   waitForPageFontsLoaded,
   waitForSelector
 } from '../../common/utils';
 import type { MockCallAdapterState } from '../../../common';
 
+/* @conditional-compile-remove(call-readiness) */
```
Comment (PorterNan) [inline packages/react-composites/tests/browser/call/hermetic/CallReadiness.test.ts:16]: Nice! How do you figure out all these feature tags? I can imagine it requires lots of code reading (But we don't have to do it any more after your PR!)

**Review by PorterNan — APPROVED** [review]

File: packages/react-composites/scripts/runBrowserTests.mjs (hunk 1 above)
Comment (prprabhu-ms) [reply packages/react-composites/scripts/runBrowserTests.mjs:180]: The intention is to keep moving logic from `package.json` `scripts` object to actual node scripts.

We can extract common logic as we use it.

File: packages/react-composites/tests/browser/call/hermetic/CallReadiness.test.ts (hunk 2 above)
Comment (prprabhu-ms) [reply packages/react-composites/tests/browser/call/hermetic/CallReadiness.test.ts:16]: Yeah, just my best guess followed by asking the feature owners.

**Pull request merged**